opm/input/eclipse/Schedule/Action/ASTNode.cpp

```diff
@@ -104,6 +107,52 @@ Opm::Action::ASTNode::serializationTestObject()
     return result;
 }
 
+void Opm::Action::ASTNode::add_child(const ASTNode& child)
+{
+    this->children.push_back(child);
+}
+
+Opm::Action::Result
+Opm::Action::ASTNode::eval(const Context& context) const
+{
+    if (this->empty()) {
+        throw std::invalid_argument {
+            "ASTNode::eval() should not reach leafnodes"
```

Comment:
leaf nodes

Reply:
> leaf nodes

Thanks.  I've pushed an update to fix that.